navbar-service.js

```diff
@@ -28,6 +28,15 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  // service.display may be a string or an array of strings
+  // valid values: all, brand, menu, toolbar
+  service.display = 'all';
+  service.siteTitle = null;
+
+  $rootScope.$on("$routeChangeSuccess", function() {
```

Comment:
I wonder if we need this to be more like ... you set this *for your route*. And then we let people also set it for "all routes" via `*` or something.